01/assets/js/app.js

```diff
@@ -1,4 +1,54 @@
 const a = '4.2';
 const b = 9;
 
-console.log(a, b);
\ No newline at end of file
+console.log(a, b);
+
+console.log(typeof a);
+console.log(typeof b);
+
+console.log('PRZERWA');
+
+const changedA = parseFloat(a);  //podchwytliwe męczyłem się z parseInt :D
+console.log(changedA);
+
+console.log('PRZERWA');
+
+const addition = changedA + b;    
+const subtraction = changedA - b;
+const multiplication = changedA * b;
+const division = changedA / b;
+const modulo = changedA % b;
+
+console.log(addition);
+console.log(subtraction);
+console.log(multiplication);
+console.log(division);
+console.log(modulo);
+
+// Nie do końca zrozumiałem to polecenie, mam nadzieję że jest w porządku. Jak coś to poprawię :D
+
+if (addition > 20) {
+    console.log ('Dodawanie to wynik powyżej 20');
+}
+
+else if (subtraction > 20) {
```

Comment:
Dlaczego tutaj używasz `else` raczej są to działa nie powiązane wiec użyłbym wielokrotnie samego `if`